rasa/core/actions/action.py

```diff
@@ -1154,6 +1154,8 @@ async def run(
 
         for slot in user_slots:
             for mapping in slot.mappings:
+                _check_mapping_validity(mapping, domain)
```

Comment:
Should we not skip the mapping if is invalid?
```suggestion
                if not _check_mapping_validity(mapping, domain):
                       continue
```